### PR TITLE
Link to enrollment from referral detail, style updates

### PR DESCRIPTION
**Pull request by gigxz**
Requires https://github.com/greenriver/hmis-warehouse/pull/3131
Screen Shot 2023-06-07 at 5 22 20 PM

### CODE REVIEW

File: src/modules/form/components/viewable/DynamicViewFields.tsx [hunk 1]
```diff
@@ -74,17 +75,19 @@ const DynamicViewFields: React.FC<Props> = ({
...
-        item={item}
-        value={isDisabled ? undefined : values[item.linkId]}
-        nestingLevel={nestingLevel}
-        horizontal={horizontal}
-        pickListRelationId={pickListRelationId}
-        // Needed because there are some enable/disabled and autofill dependencies that depend on PickListOption.labels that are fetched (PriorLivingSituation is an example)
-        adjustValue={itemChanged}
-        {...props}
-      />
+      <Grid item>
```
Comment (ttoomey) [inline src/modules/form/components/viewable/DynamicViewFields.tsx]: The expectation is that this is always rendered inside DynamicView which provides the `<Grid container>`?

File: src/modules/form/components/viewable/DynamicViewFields.tsx [hunk 2]
```diff
@@ -74,17 +75,19 @@ const DynamicViewFields: React.FC<Props> = ({
...
+        <DynamicViewField
+          key={item.linkId}
```
Comment (ttoomey) [inline src/modules/form/components/viewable/DynamicViewFields.tsx]: key probably wants to live up a level on `<Grid item>`

**Review by ttoomey — APPROVED** [review]
Looks good!

File: src/modules/form/components/viewable/DynamicViewFields.tsx (hunk 1 above)
Comment (gigxz) [reply src/modules/form/components/viewable/DynamicViewFields.tsx]: Yeah, exactly.

File: src/modules/form/components/viewable/DynamicViewFields.tsx (hunk 2 above)
Comment (gigxz) [reply src/modules/form/components/viewable/DynamicViewFields.tsx]: Whoops, thanks.

**Pull request merged**